u2f-ref-code/java/src/com/google/u2f/key/messages/TransferAccessMessage.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * 
+ */
+package com.google.u2f.key.messages;
+
+import java.io.ByteArrayInputStream;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+import java.util.Objects;
+
+import com.google.u2f.U2FException;
+
+/**
+ * @author at
+ */
+public class TransferAccessMessage {
+  private final int sequenceNumber;
+  private final byte[] newUserPublicKey;
+  private final byte[] applicationParameter;
+  private final X509Certificate attestationCertificate;
+  private final byte[] signatureAuthenticationKey;
```

Comment:
s/signatureAuthenticationkey/signatureUsingAuthenticationKey/

same for next var